sdk/identity/identity/package.json

```diff
@@ -20,14 +22,19 @@
     "./dist-esm/src/credentials/usernamePasswordCredential.js": "./dist-esm/src/credentials/usernamePasswordCredential.browser.js",
     "./dist-esm/src/credentials/azurePowerShellCredential.js": "./dist-esm/src/credentials/azurePowerShellCredential.browser.js",
     "./dist-esm/src/util/authHostEnv.js": "./dist-esm/src/util/authHostEnv.browser.js",
+    "./dist-esm/src/util/isNode.js": "./dist-esm/src/util/isNode.browser.js",
```

Comment:
Elsewhere in the package I see `import { isNode } from "@azure/core-utl"`, so do we have to have our own isNode in addition to that?

Reply:
Thanks! I forgot to clean this up.